lib/pacemaker/pcmk_sched_actions.c

```diff
@@ -1413,15 +1435,8 @@ check_action_definition(pe_resource_t * rsc, pe_node_t * active_node, xmlNode *
              * re-scheduled so the next run uses the new parameters.
              * The old instance will be cancelled automatically.
              */
-            pe_action_t *op = NULL;
-
-            trigger_unfencing(rsc, active_node,
-                              "Device parameters changed (reschedule)", NULL,
-                              data_set);
             crm_log_xml_debug(digest_data->params_all, "params:reschedule");
```

Comment:
I think it makes sense to call crm_log_xml_debug in the function as well.  This is what's being done with force_restart.

Reply:
makes sense

Reply:
actually it makes more sense to move the debug out of force_restart(), b/c ReloadRsc() is recursive and can't easily do the same